routers/repo/view.go

```diff
@@ -265,6 +265,17 @@ func renderFile(ctx *context.Context, entry *git.TreeEntry, treeLink, rawLink st
 			ctx.Data["RawFileLink"] = fmt.Sprintf("%s%s.git/info/lfs/objects/%s/%s", setting.AppURL, ctx.Repo.Repository.FullName(), meta.Oid, filenameBase64)
 		}
 	}
+	// Check LFS Lock
+	lfsLock, err := ctx.Repo.Repository.GetTreePathLock(ctx.Repo.TreePath)
```

Comment:
We should probably also not check this if file is not LFS file

Reply:
@lafriks 
As @davidsvantesson mentioned in #8687, `git lfs can be used without storing the file as a git lfs file`. Therefore, a file with LFS lock may not have `LFSMetaObject` and it's `ctx.Data["IsLFSFile"]` will be false.

So it seems to me that we should always check if a file has LFSLock. If there is anything I misunderstand, please let me know. 😄

Reply:
but I could still push changes to that file from git client and LFS lock would be ignored imho, or am I wrong?

Reply:
It's dependent on the status of the lockable attribute - which can be in .gitattributes as well as .git/info/attributes (so you can't just rely on the contents of .gitattributes.)

I think it's one of those things whereby you could add a simple marker to the locked files to show that they're not marked by .gitattributes?

I'll think about it and consider adding a marker to #8726